### ISSUE TITLE
Changing view from Concepts to Shots gives an error

### ISSUE THREAD

**Issue by johhnry** [cgwire/kitsu#1527]
**Context**

Studio name: Welcome To Love
Kitsu version: 0.19.55
Production type: Only Shots

**Describe the bug**

If I go to the `Concepts` page then to the `Shots` page, I get this error: `An error occurred while loading data.`

And in the dev console:

```
TypeError: ft is undefined
    l shots.js:865
    LOAD_SHOTS_END shots.js:858
    registerMutation vuex.esm.js:844
    commit vuex.esm.js:466
    commit vuex.esm.js:465
    _withCommit vuex.esm.js:624
    commit vuex.esm.js:464
    commit vuex.esm.js:409
    loadShots shots.js:427
    promise callback*loadShots shots.js:425
    registerAction vuex.esm.js:851
    dispatch vuex.esm.js:516
    dispatch vuex.esm.js:406
    te vuex.esm.js:1064
    currentSection Shots.vue:1164
    VueJS 11
    init vue-router.esm.js:3005
    init vue-router.esm.js:3004
    updateRoute vue-router.esm.js:2414
    transitionTo vue-router.esm.js:2263
    confirmTransition vue-router.esm.js:2402
    ee vue-router.esm.js:2084
    ee vue-router.esm.js:2091
    runQueue vue-router.esm.js:2095
    confirmTransition vue-router.esm.js:2397
    ee vue-router.esm.js:2084
    ee vue-router.esm.js:2088
    xe vue-router.esm.js:2384
    resolveAsyncComponents vue-router.esm.js:2162
    xe vue-router.esm.js:2362
    ee vue-router.esm.js:2087
    ee vue-router.esm.js:2091
    ee vue-router.esm.js:2091
    ee vue-router.esm.js:2091
    runQueue vue-router.esm.js:2095
    confirmTransition vue-router.esm.js:2392
    transitionTo vue-router.esm.js:2260
    push vue-router.esm.js:2606
    push vue-router.esm.js:3039
shots.js:442:16
```

**Screenshots**

![image](https://github.com/user-attachments/assets/a9c8588d-cbb4-4ca8-b1cd-e28230c9b971)

**Desktop (please complete the following information):**

 - OS: Windows
 - Browser: Firefox
 - Version: 129.0.2

**Comment by NicoPennec** [comment]
Hi @johhnry,
Thank you for the bug report.
I'm reproducing your issue with an "only shots" production.
We will investigate this matter.

**Comment by johhnry** [comment]
@NicoPennec thanks, just to mention that it does not seem to depend on the production type. I just tested with a `Short` production and the bug is still there.

**Comment by NicoPennec** [comment]
Fixed on Kitsu >= 0.19.56
